numpy/_core/tests/test_array_coercion.py

```diff
@@ -46,7 +46,7 @@ class _SequenceLike:
         def __len__(self):
             raise TypeError
 
-        def __getitem__(self):
+        def __getitem__(self):  # noqa: PLE0302
```

Comment:
might as well do

```suggestion
        def __getitem__(self, _, /):
```

because it'll `raise TypeError` either way